meson.build

```diff
@@ -54,10 +54,13 @@ if not ftdi_dep.found()
 endif
 
 gpiod_dep = dependency('libgpiod', required: server_opt)
+jpeg_dep = dependency('libjpeg', required: server_opt)
```

Comment:
given this requires a non-obvious library, it would be nice to make it opt-outable (default=y)

Reply:
@RayyanAnsari is this point addressed?

Reply:
Make the dependency optional if it is not present, stub the camera.c out.